org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -407,6 +464,26 @@ package lang {
 		override method asSet() = self
 
 		override method anyOne() native
+
+		/**
+		 * Returns a new Set with the elements of both self and another collection.
+		 * @returns a Set
+		 */
+		 method union(another) = self + another
```

Comment:
> I am not sure but maybe + is also implemented for lists as concatenation?

That means that you can say to a list to union another list, and you will get the concatenation of both lists, right? That is weird.

> Yes, it's a synonym.
Ok. I believe that synonyms are a big headache for maintainers - _which method should I override?_ - and students - _which methods should I use?_.

Actually, in another issue I am requesting to remove the equal/== synonym.  

What about just defining `+` in collections, but `union` just in sets? If we are going to the _union just works for sets_ approach, naming a concatenation as `union` is not a good idea.